appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/BaseContainer.java

```diff
@@ -234,33 +203,30 @@
     // constants for EJB(Local)Home/EJB(Local)Object methods,
     // used in authorizeRemoteMethod and authorizeLocalMethod
     private static final int EJB_INTF_METHODS_LENGTH = 16;
-    static final int EJBHome_remove_Handle      = 0;
-    static final int EJBHome_remove_Pkey        = 1;
-    static final int EJBHome_getEJBMetaData     = 2;
-    static final int EJBHome_getHomeHandle      = 3;
-    static final int EJBLocalHome_remove_Pkey   = 4;
-    static final int EJBObject_getEJBHome       = 5;
+    static final int EJBHome_remove_Handle = 0;
+    static final int EJBHome_remove_Pkey = 1;
+    static final int EJBHome_getEJBMetaData = 2;
+    static final int EJBHome_getHomeHandle = 3;
+    static final int EJBLocalHome_remove_Pkey = 4;
+    static final int EJBObject_getEJBHome = 5;
     protected static final int EJBObject_getPrimaryKey = 6; //TODO - move related to entity-container
-    static final int EJBObject_remove           = 7;
-    static final int EJBObject_getHandle        = 8;
-    static final int EJBObject_isIdentical      = 9;
+    static final int EJBObject_remove = 7;
+    static final int EJBObject_getHandle = 8;
+    static final int EJBObject_isIdentical = 9;
     static final int EJBLocalObject_getEJBLocalHome = 10;
     protected static final int EJBLocalObject_getPrimaryKey = 11; //TODO - move related to entity-container
-    static final int EJBLocalObject_remove      = 12;
+    static final int EJBLocalObject_remove = 12;
     static final int EJBLocalObject_isIdentical = 13;
-    static final int EJBHome_create             = 14;
-    static final int EJBLocalHome_create        = 15;
+    static final int EJBHome_create = 14;
+    static final int EJBLocalHome_create = 15;
 
     // true if home method, false if component intf method.
     // Used for setting info on invocation object during authorization.
-    private static final boolean[] EJB_INTF_METHODS_INFO =
-    { true,  true,  true,  true,  true,
-      false, false, false, false, false,
-      false, false, false, false,
-      true,  true };
+    private static final boolean[] EJB_INTF_METHODS_INFO = { true, true, true, true, true, false, false, false, false, false, false, false,
```

Comment:
I dislike previous array of booleans as I have no idea what it is, and this is probabaly even worse. But ok, the biggest issue is that it is an array of 16 booleans.